bigbluebutton-html5/imports/ui/components/breakout-room/component.jsx

```diff
@@ -159,20 +164,29 @@ class BreakoutRoom extends PureComponent {
       };
     return (
       <div className={styles.breakoutActions}>
-        <Button
-          label={intl.formatMessage(intlMessages.breakoutJoin)}
-          aria-label={`${intl.formatMessage(intlMessages.breakoutJoin)} ${number}`}
-          onClick={() => {
-            this.getBreakoutURL(breakoutId);
-            logger.debug({
-              logCode: 'breakoutroom_join',
-              extraInfo: { logType: 'user_action' },
-            }, 'joining breakout room closed audio in the main room');
-          }
-          }
-          disabled={disable}
-          className={styles.joinButton}
-        />
+        {isUserInBreakoutRoom(breakoutId)
+          ? (
+            <span className={styles.alreadyConnected}>
+              {intl.formatMessage(intlMessages.alreadyConnected)}
+            </span>
+          )
+          : (
+            <Button
+              label={intl.formatMessage(intlMessages.breakoutJoin)}
+              aria-label={`${intl.formatMessage(intlMessages.breakoutJoin)} ${number}`}
+              onClick={() => {
+                this.getBreakoutURL(breakoutId);
+                logger.debug({
+                  logCode: 'breakoutroom_join',
+                  extraInfo: { logType: 'user_action' },
+                }, 'joining breakout room closed audio in the main room');
+              }
+              }
+              disabled={disable}
+              className={styles.joinButton}
+            />
+          )
+        }
         {
           moderatorJoinedAudio
             ? [
```

Comment:
Unrelated to this PR, but do you know why this is a square bracket? I just noticed it when reading through the code.

Reply:
Because we render two things when the user is connected in audio, the `|` separator and the button to join audio.

Reply:
Ah true. React throws up a unique key warning for this component with the array I think. I know it's unrelated, but could you add a key to the Button just to satisfy warning.
```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `BreakoutRoom`. See https://fb.me/react-warning-keys for more information.
    in Button (created by BreakoutRoom)
    in BreakoutRoom (created by InjectIntl(BreakoutRoom))
    in InjectIntl(BreakoutRoom) (created by BreakoutContainer)
    in BreakoutContainer (created by ReactMeteorDataComponent)
    in ReactMeteorDataComponent (created by PanelManager)
    in div (created by PanelManager)
    in PanelManager (created by InjectIntl(PanelManager))
    in InjectIntl(PanelManager) (created by App)
    in section (created by App)
    in main (created by App)
    in App (created by InjectIntl(App))
    in InjectIntl(App) (created by AppContainer)
    in AppContainer (created by ReactMeteorDataComponent)
    in ReactMeteorDataComponent (created by ModalMounterWrapper)
    in ModalMounterWrapper (created by InjectIntl(ModalMounterWrapper))
    in InjectIntl(ModalMounterWrapper) (created by Base)
    in IntlProvider (created by IntlStartup)
    in IntlStartup (created by Base)
    in Base (created by ReactMeteorDataComponent)
    in ReactMeteorDataComponent
    in Subscriptions (created by ReactMeteorDataComponent)
    in ReactMeteorDataComponent
    in AuthenticatedHandler
    in JoinHandler
```